### ISSUE TITLE
media-sound/serialosc: Don't use git-r3 for versioned ebuilds

### ISSUE THREAD

**Issue by NexAdn** [gentoo-audio/audio-overlay#517]
We should only use git-r3 for the live ebuild, since this allows Manifest checksum generation, distfile caching and downloading the distfiles before building the software.
This might involve some extra work, since upstream facilitates git submodules.
However, it is doable.

**Comment by simonvanderveldt** [comment]
Yeah, I believe I did it this way because the upstream source release are just the GitHub generated ones and they unfortunately don't include the git submodule sources.
We could ask upstream and/or create a PR to upstream to make sure releases have an artifact that contains all the relevant sources.

**Comment by NexAdn** [comment]
Actually both submodules are on GitHub, so we can just download tarballs of the respective submodule commits and unpack/link them in the right place in WORKDIR.
